code/modules/mob/dead/observer/verbs.dm

```diff
@@ -238,10 +238,12 @@
 	set name = "Toggle Darkness"
 	set category = "Ghost"
 
-	if (see_invisible == SEE_INVISIBLE_OBSERVER_NOLIGHTING)
-		see_invisible = SEE_INVISIBLE_OBSERVER
-	else
-		see_invisible = SEE_INVISIBLE_OBSERVER_NOLIGHTING
+	if (client && client.darkness_planemaster)
+		switch(client.darkness_planemaster.alpha)
+			if(255)		client.darkness_planemaster.alpha = 230
```

Comment:
Is this state really necessary, especially since the only below it does the exact same thing

Reply:
what do you mean? you'd like only one intermediate darkness state?

Reply:
Yeah, I don't see a reason to have two

Reply:
i put two because i'm not sure how the darkness will look on other machines, on my current shitty laptop i can see no problem with the darker of the two, but on my main monitor it's still very dark